components/region_cache_memory_engine/src/read.rs

```diff
@@ -497,7 +521,10 @@ impl Iterator for RangeCacheIterator {
         };
 
         self.seek_for_prev_internal(&seek_key);
-        self.collects_stats();
+        if self.valid {
+            self.local_stats.bytes_read += (self.key().len() + self.value().len()) as u64;
+            self.local_stats.number_db_seek_found += 1;
```

Comment:
I think the block of update local_stats can be put in one function, even though the number_xxx is different. One way is to have the mutable reference parameter for number_xxx